### ISSUE TITLE
Community Settings Page - Form elements must have labels

### ISSUE THREAD

**Issue by fenekku** [inveniosoftware/invenio-communities#1075]
Package version (if known): v11 up

## Describe the bug

There is a critical WCAG 2.1 AA violation on the community settings page ('/communities/<any test community>/settings'):

**in the page**

![form_elements_must_have_labels_in_situ](https://github.com/inveniosoftware/invenio-communities/assets/1932651/8e08545c-b9b1-46d5-9d77-e1df2ed177ef)

**description from Axe devtool**

![form_elements_must_have_labels](https://github.com/inveniosoftware/invenio-communities/assets/1932651/7259b11c-c976-4ad1-af61-da5204d37a2e)

Unfortunately this may be a wontfix issue. The problem is essentially on React-Semantic-UI side (see
https://github.com/Semantic-Org/Semantic-UI-React/issues/4349 and some others more general): https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/assets/semantic-ui/js/invenio_communities/settings/profile/CommunityProfileForm.js#L403 does the right thing and passes the label with the appropriate `for` property. It's the react semantic ui that doesn't place the ARIA/accessible attributes properly.

A potential solution could be to implement our own accessible dropdown like we did on the record landing page, but we would need to wire it up with the form field system and that seems really daunting.

So maybe this is a ticket to just keep track of that problem.

**Comment by gutumd** [comment]
Hello there, I've faced same issue recently with the Dropdown component, and that's how I've solved it:
https://codesandbox.io/p/sandbox/semantic-ui-example-forked-2mvnx6?file=%2Fexample.js%3A38%2C24

![image](https://github.com/inveniosoftware/invenio-communities/assets/86073097/130678c3-793f-410e-8b10-d87438586325)
